Tests/SentryTests/Integrations/Performance/UIViewController/SentryUIViewControllerSwizzlingTests.swift

```diff
@@ -56,6 +63,20 @@ class SentryUIViewControllerSwizzlingTests: XCTestCase {
         super.tearDown()
         clearTestState()
     }
+    
+    func testExternalViewControllerImage() {
+        //Test to ensure ExternalUIViewController exists in an external lib
+        //just in case someone changes the settings of the `SentryTestUtils` lib
+        let imageName = String(
+            cString: class_getImageName(SentryUIViewControllerSwizzlingTests.self)!,
+            encoding: .utf8)! as NSString
+        
+        let externalImageName = String(
+            cString: class_getImageName(ExternalUIViewController.self)!,
+            encoding: .utf8)! as NSString
+        
+        XCTAssertNotEqual(externalImageName, imageName)
```

Comment:
`l`: I think we could add an assertion failure message explaining what the problem is.